composer.json

```diff
@@ -57,7 +57,6 @@
         "symfony/phpunit-bridge": "^2.8.7 || ^3.0",
         "sensio/framework-extra-bundle": "~3.0",
         "symfony/monolog-bundle": "^2.8.7 || ^3.0",
-        "twig/twig": "~1.11",
```

Comment:
Since we are directly using twig, we still should list it in here, instead of relying on another package to require it.

Reply:
twig was required twice (in require and require-dev).